File: Priprema K3 - Predavanje/IN/Python/roba.py
'''
Created on Dec 25, 2018

@author: Marko
'''

class Roba:

    def __init__(self, marka = "Nike", naziv = "patike", tezinaKG = 1):
        self.marka = marka
        self.naziv = naziv
        self.tezinaKG = tezinaKG
    
    def __str__(self):
        return "(\"" + self.marka + "\", \"" + self.naziv + "\", " + str(self.tezinaKG) + ")"        

File: Priprema K3 - Predavanje/IN/Java/Roba.java
public class Roba {
	private String marka = "Nike";
	private String naziv = "patike";
	private int tezinaKG = 1;
	
	public String getMarka() {
		return marka;
	}

	public String getNaziv() {
		return naziv;
	}

	public int getTezinaKG() {
		return tezinaKG;
	}
	
	public Roba() {
	}

	public Roba(String marka, String naziv, int tezinaKG) {
		this.marka = marka;
		this.naziv = naziv;
		this.tezinaKG = tezinaKG;
	}

	@Override
	public String toString() {
		return "(\"" + this.marka + "\", \"" + this.naziv + "\", " + this.tezinaKG + ")";       
	}
}
